File: leetcode/1easy/이경호/3206.cpp
class Solution {
  public:
      int numberOfAlternatingGroups(vector<int>& colors) {
          int ans = 0;
          colors.insert(colors.end(), colors.begin(), colors.end());
          for(int i = 2; i < (colors.size() / 2) + 2; i++) {
              if(colors[i - 2] != colors[i - 1] && colors[i - 1] != colors[i]) {
                  ans++;
              }
          }
          return ans;
      }
  };

File: leetcode/1easy/김민진/3206.py
<<<<<<< HEAD
class Leetcode.이재훈.이재훈.Solution:
=======
class Leetcode.이재훈.Solution:
>>>>>>> ab7f46f73d8f18bcd38c5cf06ecabae64f53a424
    def numberOfAlternatingGroups(self, colors: List[int]) -> int:
        count = 0
        for i in range(len(colors)):
            if colors[i-1] != colors[i] and colors[i] != colors[(i+1)%len(colors)]:
                count += 1
        return count
        